Signal/src/ViewControllers/DebugUI/DebugUIMessages.m

```diff
@@ -760,31 +780,105 @@ + (NSString *)randomText
+                case 0: {
+                    TSIncomingMessage *message =
+                        [[TSIncomingMessage alloc] initWithTimestamp:[NSDate ows_millisecondTimeStamp]
+                                                            inThread:thread
+                                                            authorId:@"+19174054215"
+                                                      sourceDeviceId:0
+                                                         messageBody:randomText];
+                    [message markAsReadWithTransaction:transaction sendReadReceipt:NO updateExpiration:NO];
+                    break;
+                }
+                case 1: {
+                    TSOutgoingMessage *message =
+                        [[TSOutgoingMessage alloc] initWithTimestamp:[NSDate ows_millisecondTimeStamp]
+                                                            inThread:thread
+                                                         messageBody:randomText];
+                    [message saveWithTransaction:transaction];
+                    break;
+                }
+                case 2: {
+                    TSAttachmentPointer *pointer =
+                        [[TSAttachmentPointer alloc] initWithServerId:237391539706350548
+                                                                  key:[self createRandomNSDataOfSize:64]
+                                                               digest:nil
+                                                          contentType:@"audio/mp3"
+                                                                relay:@""
+                                                       sourceFilename:@"test.mp3"
+                                                       attachmentType:TSAttachmentTypeDefault];
+                    [pointer saveWithTransaction:transaction];
+                    TSIncomingMessage *message =
+                        [[TSIncomingMessage alloc] initWithTimestamp:[NSDate ows_millisecondTimeStamp]
+                                                            inThread:thread
+                                                            authorId:@"+19174054215"
+                                                      sourceDeviceId:0
+                                                         messageBody:nil
+                                                       attachmentIds:@[
+                                                           pointer.uniqueId,
+                                                       ]
+                                                    expiresInSeconds:0];
+                    [message markAsReadWithTransaction:transaction sendReadReceipt:NO updateExpiration:NO];
+                    break;
+                }
+                case 3: {
+                    OWSDisappearingMessagesConfiguration *configuration =
+                        [OWSDisappearingMessagesConfiguration fetchObjectWithUniqueID:thread.uniqueId
```

Comment:
Not super concerned since this is debug code, but looks like we only respect the disappearing messages configuration 1 in 4 times? Can you give me a hint as to why?